### PR TITLE
Start to add generation 9 Pokemon

### PR DESCRIPTION
**Pull request by thib3113**
I start to add gen 9 . 

I only add the names I found ( I found [Korean](https://wiki.52poke.com/wiki/%E5%AE%9D%E5%8F%AF%E6%A2%A6%E5%88%97%E8%A1%A8%EF%BC%88%E6%8C%89%E5%85%A8%E5%9B%BD%E5%9B%BE%E9%89%B4%E7%BC%96%E5%8F%B7%EF%BC%89) but seems to use a different order ) . 

Also, 980 and 987 seems missing from lists, so I let a blank line . 

Don't hesitate to add names (or correct), I will update the PR with them

### CODE REVIEW

**Comment by thib3113** [comment]
Also, letting a blank line will be a problem, because of the getId function ... 

but I think adding a false line is not a solution ... 
I think using object (with key the id of the pokemon, and the value the name of the pokemon) can be a good option (can be done really fast)

**Comment by pedrommcarrasco** [comment]
Hi @sindresorhus 👋

Could we try to move forward with this? Is there anything that you need help with?

File: data/de.json [hunk 1]
```diff
@@ -1,907 +1,1012 @@
 [
-	"Bisasam",
```
Comment (sindresorhus) [inline data/de.json:2]: Don't change the indentation.

**Comment by sindresorhus** [comment]
You need to add all the data for this to be mergable.

File: data/de.json [hunk 2]
```diff
@@ -903,5 +903,125 @@
...
+    "Beatori",
+    "Ogerpon",
+    "Briduradon",
+    "Hydrapfel",
+    "Keilflamme",
+    "Furienblitz",
+    "Eisenfels",
+    "Eisenhaupt",
+    "Terapagos",
+    "Infamomo"
```
Comment (sindresorhus) [inline data/de.json:1026]: incorrect indentation

Comment (thib3113) [reply data/de.json]: sorry ...

**Pull request merged**